src/components/heading/heading.spec.js

```diff
@@ -202,11 +231,11 @@ describe("Heading", () => {
   describe("when the backLink is a function", () => {
     it("sets it as the link onClick prop", () => {
       const backLinkSpy = jasmine.createSpy(),
-        wrapper = shallow(<Heading title="Test" backLink={backLinkSpy} />),
-        link = wrapper.find('[data-element="back"]');
+        wrapper = mount(<Heading title="Test" backLink={backLinkSpy} />),
+        link = wrapper.find('[data-element="back"]').at(0);
 
       expect(link.props().onClick.toBeDefined);
```

Comment:
I realise you didn't add this line, but since you seem to be fixing up tests anyway...

looks like `toBeDefined` is a function, so we need to call it: `.toBeDefined()`.